Bio/SeqIO/__init__.py

```diff
@@ -1081,15 +1066,9 @@ def convert(in_file, in_format, out_file, out_format, alphabet=None):
     GTTGCTTCTGGCGTGGGTGGGGGGG
     <BLANKLINE>
     """
-    if in_format in _BinaryFormats:
-        in_mode = "rb"
-    else:
-        in_mode = "rU"
-
-    if out_format in _BinaryFormats:
-        out_mode = "wb"
-    else:
-        out_mode = "w"
+    in_mode = "rb" if in_format in _BinaryFormats else "rU"
```

Comment:
I believe `"rU"` should be `"r"`.

Reply:
Good catch, yes - Python 3 has done away with universal mode (it is the default in text mode).

Reply:
There are other cases of `"rU"` to replace in the codebase too